demo-node/entrypoint-noingest.bash

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+export TENZIR_AUTOMATIC_REBUILD="${TENZIR_AUTOMATIC_REBUILD:-0}"
+export TENZIR_ALLOW_UNSAFE_PIPELINES=true
+
+# Forward signals as SIGTERM to all children.
+trap 'trap " " SIGTERM; kill 0; wait' SIGINT SIGTERM
+
+coproc NODE { exec tenzir-node --print-endpoint --commands="web server --mode=dev"; }
+# shellcheck disable=SC2034
+read -r -u "${NODE[0]}" DUMMY
+
+# Reset the verbosity so the logs don't get spammed by dozens
+# of Tenzir processes.
+export TENZIR_CONSOLE_VERBOSITY="${TENZIR_CLIENT_CONSOLE_VERBOSITY:-"info"}"
+
+# Wait until the HTTP endpoint is listening.
+while ! timeout 100 bash -c "echo > /dev/tcp/127.0.0.1/5160"; do
+  echo "Waiting for the HTTP service..."
+  sleep 1
+done
+
+nic_pipe="from nic eth0 | drop data | import"
```

Comment:
Where do we provide capture capabilities to the container?

Reply:
There is no need to. `pcap.packet` data does show up in the database, unfortunately at a very low rate, our app traffic seems to take some other path. Unfortunately `shell "ip -j a"` does not work in fargate.